packages/emulator/core/src/authEndpoints.ts

```diff
@@ -57,6 +57,6 @@ export const v32Authentication = {
 };
 
 export const speech = {
-  // Access token for Bing Speech Api
-  tokenEndpoint: 'https://login.botframework.com/v3/speechtoken',
+  // Access token for Cognitive Services Api
+  tokenEndpoint: 'https://login.botframework.com/v3/speechtoken/speechservice',
```

Comment:
The official name for the service is "Cognitive Services Speech Services". So I believe this endpoint should be changed.

```suggestion
  tokenEndpoint: 'https://login.botframework.com/v3/speechtoken/speechservices',
```